### PR TITLE
HA: Create pacemaker resources for openstack_wsgi (bsc#1016955)

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by dirkmueller** [comment]
[bsc#1016955 - keystone barclamp doesn't create apache pacemaker resources](https://trello.com/c/62hGljSj/241-bsc-1016955-keystone-barclamp-doesn-t-create-apache-pacemaker-resources)

**Comment by rhafer** [comment]
It seems  crowbar-pacemaker::apache needs```node[:apache][:listen_ports_crowbar]``` to be set. At least keystone, ceilometer and aodh don't to that currently.

File: chef/cookbooks/aodh/recipes/aodh.rb [hunk 1]
```diff
@@ -105,6 +105,9 @@
 aodh_protocol = node[:aodh][:api][:protocol]
 aodh_port = node[:aodh][:api][:port]
 
+node.normal[:apache][:listen_ports_crowbar] ||= {}
+node.normal[:apache][:listen_ports_crowbar][:aodh] = { plain: [aodh_port] }
```
Comment (vuntz) [inline chef/cookbooks/aodh/recipes/aodh.rb]: This needs to be `bind_port`

File: chef/cookbooks/keystone/recipes/ha.rb [hunk 2]
```diff
@@ -89,4 +89,6 @@
   end
 
   crowbar_pacemaker_sync_mark "create-keystone_ha_resources"
+elsif node[:keystone][:frontend] == "apache"
+  include_recipe "crowbar-pacemaker::apache"
```
Comment (vuntz) [inline chef/cookbooks/keystone/recipes/ha.rb]: As a sidenote: we should should really remove the support for other frontends...

Comment (dirkmueller) [inline chef/cookbooks/keystone/recipes/ha.rb]: agreed, will do so later..

**Comment by vuntz** [comment]
I re-triggered gating...

**Comment by jsuchome** [comment]
Could we dare to merge this? Or should we really (as the change is HA related) wait to see hacloud=1 gating green?

(I hope this PR could turn the upgrade job green)

**Comment by nicolasbock** [comment]
@jsuchome I suspect that the HA gating failure is due to the same problem I am working on in [this card](https://trello.com/c/nDmOxH75/304-tempest-scenario-test-volume-boot-pattern-testvolumebootpatternv2-test-volume-boot-pattern-failing-in-ha). In other words, it doesn't look to me like it's related to this PR.

**Comment by jsuchome** [comment]
@dirkmueller @vuntz any chance we could merge this, then?

**Comment by vuntz** [comment]
I'd be fine merging, but I think Dirk might not want new things in until M8 is out...

**Comment by dirkmueller** [comment]
I'll merge it after rebasing

**Pull request merged**